docs/guide/static-module-assets.md

```diff
@@ -14,24 +14,32 @@ The `public` folder of each module solves a different problem from [`ui/public`]
 
 Files we place in the `public` subdirectory of any module are always deployed such that we can write URLs like this in our CSS or SCSS files:
 
-`/modules/my-module/images/bg.png`
+`/modules/custom-module/images/bg.png`
 
 Apostrophe will automatically fix these "asset paths" so they refer to the final URL of the asset, *no matter what our production environment looks like.* In addition, in production deployments, the final URL will always contain a "release identifier" so that any static assets in the browser cache from a previous release are not reused.
 
+::: warning
+If you are configuring an Apostrophe core module or other module installed via npm, and you choose to add a `public` subdirectory for that module at project level, you will need to prefix the module name with `my-` when creating CSS URLs to those assets. Note that the `my-` prefix goes in the module name part, not the organization name part.
```

Comment:
> when creating CSS URLs

My run-in with this was not in CSS. Unless I'm missing something we shouldn't describe this narrowly. We could say "when creating URLs to those assets. This also applies to other contexts described below."

Then the "Note that..." final sentence could be the first of the next paragraph.